### ISSUE TITLE
Add function to start writing to new log file at will

### ISSUE THREAD

**Issue by RhapsodySL** [marzent/IINACT#82]
As of right now IINACT will utilize the same log file for a day, regardless of what occurs within the log file. This is fine, but after a while you end up with a large log file (I’ve had mine get up to around 100MB easily). If you’re then using FFLogs to parse this, you have to upload that surplus of data, which is both harsh on slower connections and wasteful to upload to FFLogs as most people probably don’t want the entirety of their fight history for the day uploaded, particularly if the majority of that isn’t even instanced content.

I’m essentially asking for the ability to start writing to a new log at will so that when I start raiding for the day, I have a clean log file to upload to FFLogs that *only* contains the pertinent logs. The only way to do this right now is to close the client and manually delete the file since the file is locked until the game is closed. 

As an aside, an option to only have IINACT print to log once an instance is entered (and stop after exiting) would go a decent ways to reducing log bloat as there is limited usefulness for open world logs, which is particularly helpful given that IINACT also has no automated cleaning/archival of older logs (I had logs from mid 2023 when I looked recently and my folder was several gigabytes large if I recall correctly).

**Comment by StackBLU** [comment]
Pleeeeeeease please please please, I would love this so much. 

/endlog to force a new log to be created
/deletelog to delete the current log and start a new one

**Comment by marzent** [comment]
I can try implementing these once I have a bit of time (might not be before savage release though)

**Comment by cetlysm88** [comment]
Question, what exactly is parse filtering for?
I figured I could set it to party so I would lag less during Hunts and S ranks but I still see some players not in party getting through into my meter and still getting a fps hit.
It would be great if it could auto disable itself when not inside an instance to use less CPU

**Comment by Charlignon** [comment]
I would also like an option to start and stop parsing at will ! It's a bit unclear what `/endenc` does, considering you can't start an encounter manually.
Bonus if you could configure IINACT to only start parsing in instances. I know the plugins [Clarity in Chaos](https://github.com/meoiswa/ClarityInChaos) and [ChillFrames](https://github.com/MidoriKami/ChillFrames) do it, it could be an inspo for what code to add.
Full disclosure I only use IINACT for log files, and do not use overlays at all so I may be missing a lot of implications

**Comment by marzent** [comment]
Yeah you are right `/endenc` does signal the overlays that an encounter has ended...
Adding either feature is less an issue of how to hook it up to the game, but more an issue of how FFXIV_ACT_Plugin and OverlayPlugin work, in that they are not meant to arbitrarily start/stop parsing at will (or start a new logfile) and I would like to keep them as close to the original versions in terms of compatibility as possible.
@cetlysm88  the players which are still coming through are from OverlayPlugin, which ignores FFXIV_ACT_Plugins parse filter, does disabling writing out the log file help at all with CPU usage in hunt trains? If yes, that could be fairly easily be implemented.
